fluent-package/Rakefile

```diff
@@ -478,7 +478,7 @@ class BuildTask
       desc "Create fluent-package configuration files from template"
       task :fluent_package_config do
         fluentd_conf = "etc/#{PACKAGE_DIR}/#{SERVICE_NAME}.conf"
-        fluentd_conf_default = "opt/#{PACKAGE_DIR}/share/#{COMPAT_SERVICE_NAME}.conf.tmpl"
+        fluentd_conf_default = "opt/#{PACKAGE_DIR}/share/#{SERVICE_NAME}.conf.default"
         configs = [fluentd_conf]
```

Comment:
It seems that the .default suffix is no need to add. (as it was placed under /usr/share/)

Reply:
I see! I will fix it!

Reply:
Fixed!